pkg/test/expectations/expectations.go

```diff
@@ -368,8 +373,8 @@ func ExpectMakeNodesInitialized(ctx context.Context, c client.Client, nodes ...*
 	ExpectMakeNodesReady(ctx, c, nodes...)
 
 	for i := range nodes {
-		nodes[i].Labels[v1alpha5.LabelNodeRegistered] = "true"
-		nodes[i].Labels[v1alpha5.LabelNodeInitialized] = "true"
+		nodes[i].Labels[v1beta1.NodeRegisteredLabelKey] = "true"
+		nodes[i].Labels[v1beta1.NodeInitializedLabelKey] = "true"
```

Comment:
Do we maintain these labels now that we have NodeClaims?

Reply:
I'm not sure what you mean here. There should be no "machine migration" anymore, so we can ensure these labels will be here, especially in our test cases.